Tests/Naming/UniqidNamerTest.php

```diff
@@ -21,6 +21,9 @@ public function fileDataProvider(): array
             ['lala.mp3',      'mpga',            '/[a-z0-9]{13}.mp3/'],
             ['lala',          'mpga',            '/[a-z0-9]{13}.mpga/'],
             ['lala',          null,              '/[a-z0-9]{13}/'],
+            ['lala.0',        null,              '/[a-z0-9]{13}\\.0/'],
```

Comment:
Why `lala.0` name is returning `null` extension? I thought it was supposed to return `.0`